System/ContentFile.cpp

```diff
@@ -342,4 +357,30 @@ namespace RTE {
 		}
 		return returnSample;
 	}
-}
+
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+	void ContentFile::ReloadBitmap(const std::string &filePath, int conversionMode) {
+		const int bitDepth = (conversionMode == COLORCONV_8_TO_32) ? BitDepths::ThirtyTwo : BitDepths::Eight;
+
+		std::unordered_map<std::string, BITMAP *>::iterator bmpItr = s_LoadedBitmaps[bitDepth].find(filePath);
```

Comment:
Oh please just use auto here 😭 Iterators are basically the entire reason for auto to exist (and when new'ing things, I guess)

Also I'd prefer not to have the extra line of whitespace between the if and the bmpItr setting here, as I feel it's more explicit about the relation of the code block. But I appreciate that's a super minor thing.